apps/saleor-app-checkout/backend/payments/providers/stripe/webhookHandler.ts

```diff
@@ -91,10 +91,11 @@ export const checkoutSessionToTransactionCreateMutationVariables = async (
     }
 
     const getAmount = getTransactionAmountGetter({
+      // in Stripe, authorized means that the payment is authorized and charged
       authorized: getSaleorAmountFromInteger(charge.amount),
+      charged: getSaleorAmountFromInteger(charge.amount),
```

Comment:
This could be a bit problematic because transaction amounts in different states add up in Saleor.

What I mean is that the total transaction amount will be `authorized + charged` - what we should do is set it either as authorized **or** charged not both at the same time, because it will mess up calculating remaining amount to pay in order (the order will be overpaid). https://docs.saleor.io/docs/3.x/developer/checkout#updating-transaction

Reply:
Good catch! I actually copied it from another PR (multi tenant) and I didn't verify. My fault.